docs/source/api.rst

```diff
@@ -143,6 +145,18 @@ Connectivity and Cycles
    retworkx.digraph_find_cycle
    retworkx.chain_decomposition
 
+.. _graph-ops:
+
+Graph Operations
+----------------
+
+.. autosummary::
+   :toctree: stubs
+
+   retworkx.complement
+   retworkx.union
+   retworkx.cartesian_product
+
```

Comment:
+1 for doing this but #544 seems a better place for this.

Reply:
Yeah, I agree. I only did this here because I wanted to have a link to graph operations in the introduction tutorial.